numpy/linalg/linalg.py

```diff
@@ -1336,46 +1345,75 @@ def svd(a, full_matrices=True, compute_uv=True):
 
     .. versionadded:: 1.8.0
 
-    Broadcasting rules apply, see the `numpy.linalg` documentation for
-    details.
+    The decomposition is performed using LAPACK routine ``_gesdd``.
+
+    SVD is usually described for the factorization of a 2D matrix :math:`A`.
+    The higher-dimensional case will be discussed below. In the 2D case, SVD is
+    written as :math:`A = U S V^H``, where :math:`A=` ``a``, :math:`U=` ``u``,
+    :math:`S=` ``np.diag(s)`` and :math:`V^H=` ``vh``. `s` is then a 1D array
```

Comment:
Maybe "The singular values `s` ..." so that the sentence begins with a capital.

Reply:
ok